packages/account/src/Components/poi/idv-document-submit/idv-document-submit.tsx

```diff
@@ -1,53 +1,65 @@
 import React from 'react';
 import classNames from 'classnames';
-import { Button } from '@deriv/components';
-import { Formik } from 'formik';
-import { localize } from '@deriv/translations';
+import { Form, Formik, FormikHelpers } from 'formik';
+import { GetSettings, ResidenceList, IdentityVerificationAddDocumentResponse } from '@deriv/api-types';
+import { Button, HintBox, Text } from '@deriv/components';
+import { Localize, localize } from '@deriv/translations';
 import {
     WS,
-    getIDVNotApplicableOption,
     toMoment,
     filterObjProperties,
     isDesktop,
     removeEmptyPropertiesFromObject,
     formatIDVFormValues,
 } from '@deriv/shared';
-import { documentAdditionalError, getRegex, validate, makeSettingsRequest, validateName } from 'Helpers/utils';
-import FormFooter from 'Components/form-footer';
 import BackButtonIcon from 'Assets/ic-poi-back-btn.svg';
+import PoiNameDobExample from 'Assets/ic-poi-name-dob-example.svg';
+import FormFooter from 'Components/form-footer';
 import IDVForm from 'Components/forms/idv-form';
 import PersonalDetailsForm from 'Components/forms/personal-details-form';
 import FormSubHeader from 'Components/form-sub-header';
-import { GetSettings, IdentityVerificationAddDocumentResponse, ResidenceList } from '@deriv/api-types';
-import { TIDVFormValues, TInputFieldValues, TDocumentList } from 'Types';
+import {
+    validate,
+    makeSettingsRequest,
+    validateName,
+    shouldHideHelperImage,
+    isDocumentTypeValid,
+    isAdditionalDocumentValid,
+    isDocumentNumberValid,
+} from 'Helpers/utils';
+import { TIDVForm, TPersonalDetailsForm } from 'Types';
 
 type TIDVDocumentSubmitProps = {
-    account_settings: GetSettings;
-    getChangeableFields: () => Array<string>;
-    handleBack: React.MouseEventHandler;
+    handleBack: () => void;
     handleViewComplete: () => void;
-    is_from_external: boolean;
     selected_country: ResidenceList[0];
+    account_settings: GetSettings;
+    getChangeableFields: () => Array<string>;
 };
 
+type TIdvDocumentSubmitForm = TIDVForm & TPersonalDetailsForm;
+
 const IdvDocumentSubmit = ({
     handleBack,
     handleViewComplete,
     selected_country,
-    is_from_external,
     account_settings,
```

Comment:
As @thisyahlen-deriv mentioned above we can get this from `useSettings` hooks in `@deriv/api`